src/components/UserProfile/UserSettings/UserGeneralSettings/index.tsx

```diff
@@ -167,15 +167,17 @@ const UserGeneralSettings: React.FC = () => {
                     >
                       <option value="">
                         {intl.formatMessage(messages.languageServerDefault, {
-                          applicationTitle: currentSettings.applicationTitle,
-                          language:
-                            intl.formatDisplayName(
-                              currentSettings.originalLanguage,
-                              {
-                                type: 'language',
-                                fallback: 'none',
-                              }
-                            ) ?? currentSettings.originalLanguage,
+                          language: currentSettings.originalLanguage
+                            ? intl.formatDisplayName(
+                                currentSettings.originalLanguage,
+                                {
+                                  type: 'language',
+                                  fallback: 'none',
+                                }
+                              ) ?? currentSettings.originalLanguage
```

Comment:
I think we will need the same type of fallback here. Otherwise, this would just say `en`.